openshift-examples/keycloak.yaml

```diff
@@ -60,40 +60,34 @@ objects:
         spec:
           containers:
             - env:
-                - name: KEYCLOAK_USER
-                  value: '${KEYCLOAK_USER}'
-                - name: KEYCLOAK_PASSWORD
-                  value: '${KEYCLOAK_PASSWORD}'
-                - name: DB_VENDOR
-                  value: '${DB_VENDOR}'
-                - name: JGROUPS_DISCOVERY_PROTOCOL
```

Comment:
is it intentional to drop the clustering configuration?

In case you wanna preserve it, the new variables are:
```yaml
- name: jgroups.dns.query
  value: 'dns_query=${APPLICATION_NAME}.${NAMESPACE}.svc.cluster.local'
- name: KC_CACHE
  value: 'ispn'
- name: KC_CACHE_STACK
  value: 'kubernetes'
```
cc. @pedroigor

Reply:
Not sure if this was even working before. I don't see discovery service defined here e.g.

We're also not exposing number of replicas as a param, so we technically don't support clustering. We could of course add it but it's out of scope of this PR.